src/app/templates/_.gitignore

```diff
@@ -30,6 +30,4 @@ node_modules/
 
 # Build-related directories
 dist/
-docs/api/
-es5/
-test/dist/
+
```

Comment:
well `test/dist`, `docs/api` and `es5` will no longer exist in the root folder. All of them will be in `dist/test` or `dist/docs` or `dist/es5`